openmetadata-ui/src/main/resources/ui/src/components/Ingestion/IngestionRecentRun/IngestionRecentRun.test.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { act } from 'react-test-renderer';
+import * as APIs from '../../../axiosAPIs/ingestionPipelineAPI';
+import { getRunHistoryForPipeline } from '../../../axiosAPIs/ingestionPipelineAPI';
+import { IngestionPipeline } from '../../../generated/entity/services/ingestionPipelines/ingestionPipeline';
+import { IngestionRecentRuns } from './IngestionRecentRuns.component';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest
+    .fn()
+    .mockImplementation(() => ({ t: (key: string) => key })),
+}));
```

Comment:
We have globally mocked this so can we remove it from here?

Reply:
The change wasn't there in this PR. We might have it now So I will remove it